### PR TITLE
Dnd vue playground

### PR DESCRIPTION
**Pull request by Tepexic**
As requested in #36 

This version still has bugs when the droppable element leaves the parent container

### CODE REVIEW

File: packages/dnd/playgrounds/dflex-vue-dnd/src/components/Core.vue [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      type: String,
+      required: true,
+      default: Math.random().toString(),
+    },
+    content: {},
+  },
+
+  mounted() {
+    // Register element in store
+    store.register({ id: this.id, element: this.$refs[this.id] });
```
Comment (jalal246) [inline packages/dnd/playgrounds/dflex-vue-dnd/src/components/Core.vue:20]: registering element required depth as property: 

`store.register({ id: this.id, element: this.$refs[this.id], depth: 0 })`

File: packages/dnd/playgrounds/dflex-vue-dnd/src/components/Drag.vue [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    id: {
+      type: String,
+      required: true,
+      default: Math.random().toString(),
+    },
+  },
+
+  mounted() {
+    // Register element in store
+    store.register({ id: this.id, element: this.$refs[this.id] });
```
Comment (jalal246) [inline packages/dnd/playgrounds/dflex-vue-dnd/src/components/Drag.vue:21]: the same goes here, but with higher depth, as it is the parent to `depth:0`

`store.register({ id: this.id, element: this.$refs[this.id], depth: 1 })`

**Comment by jalal246** [comment]
would you mind adding the new changes to the [main readme](https://github.com/jalal246/dflex/tree/master/packages/dnd) like we've done before with draggable:

https://github.com/jalal246/dflex/tree/master/packages/draggable#examples

**Comment by Tepexic** [comment]
Will do!
Thanks for reviewing

**Comment by Tepexic** [comment]
Thank you!

**Comment by jalal246** [comment]
@Tepexic Thank you for your contribution. You've expanded the usage of DFlex to Vue users which is awesome. And I hope you consider more contributions by opening new issues or applying PRs.

**Comment by Tepexic** [comment]
Thank you! I'll check the issues to see if I can tackle another one!

**Pull request merged**